ArcGISRuntimeSDKQt_CppSamples/Maps/CreateDynamicBasemapGallery/CreateDynamicBasemapGallery.h

```diff
@@ -0,0 +1,92 @@
+// [WriteFile Name=CreateDynamicBasemapGallery, Category=Maps]
+// [Legal]
+// Copyright 2024 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifndef CREATEDYNAMICBASEMAPGALLERY_H
+#define CREATEDYNAMICBASEMAPGALLERY_H
+
+#include <QAbstractListModel>
+#include <QMap>
+#include <QObject>
+#include <QUrl>
+
+Q_MOC_INCLUDE("MapQuickView.h")
+Q_MOC_INCLUDE("BasemapStyleInfo.h")
+Q_MOC_INCLUDE("BasemapStyleListModel.h")
+
+class BasemapStyleListModel;
+class QAbstractListModel;
+
+namespace Esri::ArcGISRuntime {
+class BasemapStyleInfo;
+enum class BasemapStyleLanguageStrategy;
+class Map;
+class MapQuickView;
```

Comment:
I think the precedent is to break these into `class` and `enum class` groupings.

Reply:
Classes and enum classes grouped in https://github.com/Esri/arcgis-maps-sdk-samples-qt/pull/1714/commits/dee1e73e6aaf82e2a06fba77378d4d87ea5d5d58.